robyn/robyn.pyi

```diff
@@ -355,6 +355,7 @@ class WebSocketConnector:
     """
 
     id: str
+    query_params: QueryParams
```

Comment:
Hey @VishnuSanal 👋 

The PR looks amazing! Please just update the docstring of the class